html/pfappserver/root/dynamicreport/index.tt

```diff
@@ -1,4 +1,9 @@
-<h2 class="card-title">[% IF report.description ; report.description ; ELSE ; report.id ; END %]</h2>
+<div class="card-title">
+<h2>[% IF report.description ; report.description ; ELSE ; report.id ; END %]</h2>
+[% IF report.long_description %]
+<p>[% report.long_description.replace('\n', '<br>') | none %]</p>
```

Comment:
Should we really localize a user driven value ?
long_description is set by the user and isn't static

I guess that would be only good for the ones we ship by default but if that does make sense, I'll add it